### PR TITLE
edit css for affirmation carousel and arrow display

### PR DESCRIPTION
**Pull request by XiaoStevens**

### CODE REVIEW

File: app/css/Affirmation.css [hunk 1]
```diff
@@ -77,18 +76,30 @@
...
+.css-1orgm40 {
+  padding: 0px !important;
+}
+
+.-left-12 {
+  margin-left: 60px !important;
+}
+
+.-right-12 {
+  margin-right: 60px !important;
```
Comment (Shaomei) [inline app/css/Affirmation.css:104]: @XiaoStevens, what are those css classes for?

**Review by Shaomei — APPROVED** [review]
@XiaoStevens Thank you for the fixes! Thank you for also fixing the carousel.tsx component I downloaded...

Can you add the screenshot of what it looks like after the this PR just to people to better understand the difference?

I had only one comment out of curiosity. Feel free the merge after addressing my comment and adding the screenshot for "after".

File: app/css/Affirmation.css (hunk 1 above)
Comment (XiaoStevens) [reply app/css/Affirmation.css:104]: affirmation carousel
1. remove the gap between the carousel and window edges.
2. make arrows showing on the carousel panel.

Comment (XiaoStevens) [reply app/css/Affirmation.css:104]: > @XiaoStevens, what are those css classes for?

Those are for adjusting arrows' position. 
1. Remove padding for both arrows.
2,3. Adding margin to both left arrow and right arrow.

**Pull request merged**